nailgun/entities.py

```diff
@@ -1771,14 +1771,12 @@ class ContentViewVersion(
     def __init__(self, server_config=None, **kwargs):
         self._fields = {
             'content_view': entity_fields.OneToOneField(ContentView),
-            'environments': entity_fields.OneToManyField(LifecycleEnvironment),
+            'environment': entity_fields.OneToManyField(Environment),
```

Comment:
I think `LifecycleEnvironment` is correct here. Here's what I get with `Environment`:

```py
>>> cvv=ContentViewVersion(id=3).read()
nailgun.entities.ContentViewVersion(puppet_module=[], major=2, package_count=0, repositories=[nailgun.entities.Repository(id=6)], environments=[nailgun.entities.Environment(id=2), nailgun.entities.Environment(id=3)], version=u'2.0', content_view=nailgun.entities.ContentView(id=2), id=3, minor=0, description=u'My Devel content view version!')

>>> cvv.environments[0].read()
WARNING:nailgun.client:Received HTTP 404 response: {
  "error": {"message":"Resource environment not found by id '2'"}
}

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "nailgun/entity_mixins.py", line 794, in read
    attrs = self.read_json(params=params)
  File "nailgun/entity_mixins.py", line 745, in read_json
    response.raise_for_status()
  File "/usr/lib/python2.7/site-packages/requests/models.py", line 940, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://centos7-devel.virbr0.akofink-desktop//api/v2/environments/2
```

I'm pretty sure that's for puppet environments, not lifecycle environments.